integrations/sap-success-factors/helpers/utils.ts

```
@@ -0,0 +1,46 @@
+
+    // Parse SAP /Date(...)/
+    const match = sapDateString.match(/\/Date\((-?\d+)([+-]\d{4})?\)\//);
+    if (!match) {
+        throw new Error(`Invalid SAP or ISO date format: ${sapDateString}`);
+    }
+
+    const timestamp = parseInt(match[1] || '0', 10);
+    return new Date(timestamp).toISOString();
+}
+
+export function getMostRecentInfo(infos: any[] | {} | undefined) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)